### PR TITLE
CIP-0040? | Collateral output

### PR DESCRIPTION
**Pull request by SebastienGllmt**
This PR describes adding a new output type to transactions called Collateral Outputs which removes the need for wallets to explicitly set collateral (which is bad UX) and improves safety of collateral at the same time

This PR is part of a series of different ways we can improve collateral handling in Cardano. You can find the three options below

[Collateral Key](https://github.com/cardano-foundation/CIPs/pull/104)
[Collateral Output](https://github.com/cardano-foundation/CIPs/pull/216)
[Collateral Reward](https://github.com/cardano-foundation/CIPs/pull/217)

### CODE REVIEW

File: CollateralOutput/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- Restriction #1 is problematic because hardcore dApp users rarely have UTXO entries that do not contain any tokens. To combat this, wallets have created a special wallet-dependent "collateral" UTXO to reserve for usage of collateral for dApps which is not a great UX. 
+- Restriction #6 is problematic because wallets want to protect users from signing transactions with large collateral as they cannot verify whether or not the transaction will fail when submitted (especially true for hardware wallets)
+
+# Specification
+
+If phrase-2 verification fails, we can send outputs to a special output marked as the collateral output.
+
+There are two ways to create collateral outputs
+
+1. Add collateral outputs as a new field inside the transaction. This change is similar to how collateral inputs were created a new field
```
Comment (JaredCorduan) [inline CollateralOutput/README.md:40]: @SebastienGllmt this option :point_up: is actually nearly what we already have in the Babbage WIP formal spec. The only difference is that we only allow for one collateral output. See figure 1 here: https://hydra.iohk.io/build/12153630/download/1/babbage-changes.pdf

File: CollateralOutput/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 CIP: ?
 Title: Explicit Collateral Output
-Authors: Sebastien Guillemot <seba@dcspark.io>
+Authors: Sebastien Guillemot <seba@dcspark.io>, Jared Corduan <jared.corduan@iohk.io>
```
Comment (JaredCorduan) [inline CollateralOutput/README.md]:
```suggestion
Authors: Sebastien Guillemot <seba@dcspark.io>, Andre Knispel <andre.knispel@iohk.io>
```

**Review by KtorZ — APPROVED** [review]
Approving assuming that the following steps will be tackled before merging:

- Change CIP number to `40`
- Move the file into the proper `cip-0040` repository
- Add it to the README table
- Change status to `proposed`

**Comment by KtorZ** [comment]
(cc @rphair)

**Review by rphair — APPROVED** [review]
@KtorZ can other editors make changes like these (https://github.com/cardano-foundation/CIPs/pull/216#pullrequestreview-939888332) to PR assets or would @SebastienGllmt be required to do it?  If you've been doing these administrative changes before merge to other people's PR's, please let me know how 😎

**Comment by KtorZ** [comment]
@rphair it's sometimes possible depending on what the settings of the PR's authors are. Some authors disable editing of their PR, some don't. Most of the time though, we kindly ask authors to do so.

As for the how, through git usually, pushing directly on the branch (which is allowed by default for PR made to your repository, unless disabled).

**Comment by KtorZ** [comment]
@SebastienGllmt seems like this one has been forgotten a bit. If you have any time to tidy this up then please do, otherwise if you don't mind, I'll just do it :)

**Comment by rphair** [comment]
OK @KtorZ @SebastienGllmt I wanted to be sure I could do this.  Please feel free to double check & if anything wrong or missing let me know & I'll publish any fixes as a separate PR.

**Pull request merged**